### DISCUSSION TITLE
Evaluation server for performance comparison

### DISCUSSION THREAD

**Discussion by dgragnaniello** [bdd100k/bdd100k#251] [General]
Will the Evolution server be available after the competition end?

**Comment by siyuanliii** [comment]
Yes, you can still use our public [leaderboards](https://www.bdd100k.com/#leaderboards).

**Reply by pikaandhouse** [reply]
Hi,I have a question about that I made a successful submission on the MOT challenge before the deadline  and got a result ,but I forgot to choose appear on the leaderboard .When I realized that,I tried to click that option to appear on the leaderboard but failed.Could I get the certificate according to my submission result.
